src/style.css

```
@@ -1 +1,47 @@
-/* add styles here */
+body {
+margin: 0;
+padding: 0;
+}
```

Fix indents everywhere
```suggestion
body {
  margin: 0;
  padding: 0;
}
```